src/library/crate/cratestorage.h

```diff
@@ -279,6 +279,9 @@ class CrateStorage: public SqlStorage {
     static QString formatSubselectQueryForCrateTrackIds(
             CrateId crateId); // no db access
 
+    // The name must be escaped with FieldEscaper
```

Comment:
Escaping of the crate name argument and surrounding it with LIKE wildcards should be done within this function. The caller should not be aware of this. The CrateStorage class is responsible for shielding the caller from the database-dependent details.